x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -5115,6 +5115,34 @@
               }
             }
           }
+        },
+        "resources_stats": {
+          "type": "array",
+          "items": {
+            "properties": {
+              "account_id": {
+                "type": "keyword"
+              },
+              "resource_type": {
+                "type": "keyword"
+              },
+              "resource_type_doc_count": {
+                "type": "long"
```

Comment:
comment: Do we need to use [`long` or will `short`](https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html#number) work? Same comment goes for the other fields?